src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1216,6 +1218,16 @@ int sockinfo_udp::setsockopt(int __level, int __optname, __const void *__optval,
 			case IP_TOS:
 				if (__optlen <= sizeof(int)) {
 					m_tos =(uint8_t) *(int *)__optval;
+					header_tos_updater du(m_pcp);
```

Comment:
Can we move IP_TOS to sockinfo and avoid the duplication  ?

Reply:
there is a difference in behavior